src/hotel-management-app/src/app/customer/registration/registration.component.ts

```diff
@@ -24,7 +24,7 @@ export class RegistrationComponent {
 
   onSubmit() {
     if (this.registrationForm.valid) {      
-      const result = this.customerService.registerCustomer(this.registrationForm.value);
+      const result: string | void = this.customerService.registerCustomer(this.registrationForm.value);
```

Comment:
Can you remove the void since a function should either return a data or not it should be returning both based on confition

Reply:
Resolved